repositories/OptionRepository.js

```diff
@@ -7,6 +7,19 @@ const mapping = { AnswerId: "answerId" };
 const fromDb = mapFields(mapping);
 const toDb = mapFields(invert(mapping));
 
+const checkForExistingExclusive = async answerId => {
```

Comment:
We should unit test this.

Reply:
Do we need to? The pattern we've used in other places suggests that testing this in the resolvers integration test should be fine?